Refactor of contribute pages (#1437)
elixir-europe/rdmkit: pages/contribute/google_doc_way.md

---

> title: Google Doc way

Reviewer: ```suggestion
title: Google Docs way
```

toc: false
---

Follow the steps below to contribute using Google Docs:
1. Email the editorial team at [rdm-editors@elixir-europe.org](mailto:rdm-editors@elixir-europe.org)  to propose additions or updates to an existing page. Make sure you keep other contributors in CC.
2. Use our section-specific Google Doc templates to write your page:
    *  [Your task template](https://docs.google.com/document/d/11MsyGl7WJXI4dahMjbI3addEXQlpDzM19Yk_6e-gsVI)
    *  [Your domain template](https://docs.google.com/document/d/1fh8-Gq50AkXS1nhHr-gVFog-eI61dFI49nKROZlRYEQ)
    *  [Your role template](https://docs.google.com/document/d/1J8SmHw-TtVQ6P38SzucRB37upYwscUOwbTx_qllOPaI)
    *  [Tools assembly template](https://docs.google.com/document/d/1BZsc647JAmJwUkVjds5cu52_-70_TTfDMypPHqDiFrM)
    *  [National resources template](https://docs.google.com/document/d/1icB57BHMbZbxwFV9keq9ZYm5QbrDjJDeufrTdoYmHzk)

3. Notify the editorial team when you are finished.

> 4. Read feedback. The editors will assign reviewers to your page, who will provide feedback as comments on the shared Google Doc.

Reviewer 2: > Read feedback.

Read the feedback.

> 4. Read feedback. The editors will assign reviewers to your page, who will provide feedback as comments on the shared Google Doc.

Reviewer: ```suggestion
4. Read the feedback that the reviewers will provide as comments on the shared Google document.
```

> 5. Address the reviewers’ comments and let the editorial team know that you’ve finished reviewing.

Reviewer: ```suggestion
5. Address the reviewers’ comments and let the editorial team know when you are finished.
```
